.github/workflows/gradle.yml

```diff
@@ -29,9 +29,9 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
       - name: Build the Jar
-        run: './gradlew build'
+        run: './gradlew clean shadowJar --stacktrace'
       - name: Upload a Build Artifact
         uses: actions/upload-artifact@v3.1.2
         with:
-          name: 'EternalDiscordOfficer*.jar'
-          path: build/libs/EternalDiscordOfficer*.jar
\ No newline at end of file
+          name: 'Successfully build DiscordOfficer JDK${{ matrix.jdk }}'
```

Comment:
Proszę zmienić nazwę artefaktu, w przeciwnym razie będzie to wyglądać tak:
![image](https://user-images.githubusercontent.com/77227023/227630667-121c066d-0966-4bde-aa69-f9d9d7832acb.png)
a chyba nie chcemy, aby plik się tak nazywał ;)

Reply:
Nazwa ta nie jest nazwą artefaktu, lecz jedynie nazwą wyświetlaną (displayname) w archiwum zip, do którego zostanie zapakowany artefakt o kluczowej nazwie "DiscordOfficer<version>.jar".

Reply:
Nazwa ta nie jest nazwą artefaktu, lecz jedynie nazwą wyświetlaną (displayname) w archiwum zip, do którego zostanie zapakowany artefakt o kluczowej nazwie "DiscordOfficer<version>.jar".

Reply:
Masz rację, natomiast uważam że nazwa `Successfully build DiscordOfficer[...]` nie jest dobrą nazwą, nawet w przypadku takiego archiwum ZIP. Można zastosować generalną nazwę w stylu `DiscordOfficer`.

Reply:
Uzasadnij swoje stanowisko

Reply:
Nie potrzebujemy drugiego potwierdzenia, że DiscordOfficer został pomyślnie zbudowany. Można to wywnioskować z logów actionsów oraz zielonej ikonki. Konwencja nazywania zipa 'Successfully build DiscordOfficer itp.' też nie jest dobrą konwencją nazewnictwa. Raczej nie spotkałem się aby jakiś inny projekt nazywał to w ten sposób

Reply:
Kolejna sprawa jest taka, że zmienna {$matrix.jdk} nie jest prawidłowo parsowana w tej nazwie

Reply:
> Kolejna sprawa jest taka, że zmienna {$matrix.jdk} nie jest prawidłowo parsowana w tej nazwie
To trzeba to naprawić, a nie zmieniać output zip directory